cls/_zpkg/isc/sc/git/Favorites.cls

```diff
@@ -0,0 +1,55 @@
+Class %zpkg.isc.sc.git.Favorites Extends %RegisteredObject
```

Comment:
No need to extend %RegisteredObject (though you might need to add include files if you drop it)